src/utils/constants.js

```diff
@@ -28,9 +28,10 @@ export const PollTxState = {
 
 export const CHIEF = 'CHIEF';
 
-export const POLL_DEFAULT_START = getUtcDateObject();
-export const POLL_DEFAULT_END = new Date(
-  POLL_DEFAULT_START.getTime() + 7 * 24 * 60 * 60 * 1000
+const now = new Date();
+export const POLL_DEFAULT_START = getUTC(now);
+export const POLL_DEFAULT_END = getUTC(
+  new Date(now.getTime() + 7 * 24 * 60 * 60 * 1000)
 );
```

Comment:
Wrapping `POLL_DEFAULT_START` and `POLL_DEFAULT_END` with `getUTC` as above will create Date objects with the correct UTC time (floored to the minute). One caveat is that it will default to use the locale for wherever the user is located and so calling `POLL_DEFAULT_START.toUTCString()` will actually return a time equal to the users timezone offset to UTC less UTC itself. Calling `POLL_DEFAULT_START.toString()` will return the time as UTC.